INSTALL.md

```diff
@@ -1,108 +1,79 @@
 # PyBitmessage Installation Instructions
+- Binary (no separate installation of dependencies required)
+    - windows (32bit only): https://download.bitmessage.org/snapshots/
+    - linux (64bit): https://appimage.bitmessage.org/
+    - mac (64bit, not up to date): https://github.com/Bitmessage/PyBitmessage/releases/tag/v0.6.3
+- Source
+    git clone git://github.com/Bitmessage/PyBitmessage.git
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. There is also a helper script to resolve dependencies easily.
 
-PyBitmessage can be run in either one of two ways:
+Go to the directory with PyBitmessage source code and run:
+```
+python checkdeps.py
+```
+If there are missing dependencies, it will explain you what is missing and for many Unix-like systems also what you have to do to resolve it. How you then run setuptools depends on whether you want to install it to user's directory or system.
+
+# Setup Environment
+Create virtualenv with Python 2.x version
+```
+virtualenv -p python2 env
+```
+
+Activate env
+```
+source env/bin/activate
+```
 
-- straight from source
+Install requirements.txt
+```
+pip install -r requirements.txt
+```
 
- or
-- from an installed
-package.
+# Run setup tools install
+As normal user:
+```
+python setup.py install --user
+~/.local/bin/pybitmessage
+```
+or as root:
+```
+python setup.py install
+/usr/local/bin/pybitmessage
+```
 
 ## Dependencies
```

Comment:
move this to under checkdeps, say that this may help to orient if checkdeps fails